docs/cifar10_examples.md

```diff
@@ -0,0 +1,48 @@
+# CIFAR10 examples
+
+The CIFAR-10 dataset is a collection of images that are commonly used to train machine learning and computer vision algorithms. It is one of the most widely used datasets for machine learning research. The CIFAR-10 dataset contains 60,000 32x32 color images in 10 different classes. There are 6,000 images of each class. Thus, we use CIFAR10 as example to introduce using NNI in the field of computer vision.
+
+**cifar10 with NNI**
+
+This example requires pytorch. pytorch install package should be chosen based on python version and cuda version.
+
+Here is an example of the environment python==3.5 and cuda == 8.0, then using the following commands to install pytorch:
```

Comment:
We can add a link to https://pytorch.org/ to guide the user to install the pytorch.

Reply:
Thanks~ Done!